modules/core/src/main/java/org/apache/ignite/lang/ErrorGroup.java

```diff
@@ -177,10 +172,20 @@ public static int extractErrorCode(int code) {
      * @param groupCode Group code
      * @return Error Group.
      */
-    public static ErrorGroup errorGroupByCode(int groupCode) {
+    public static ErrorGroup errorGroupByGroupCode(short groupCode) {
         return registeredGroups.get(groupCode);
     }
 
+    /**
+     * Returns error group identified by the given error {@code code}.
+     *
+     * @param code Full error code
+     * @return Error Group.
+     */
+    public static ErrorGroup errorGroupByErrorCode(int code) {
```

Comment:
Same as above. ErrorCode seems to be code within group, e.g. according to
`public int registerErrorCode(short errorCode)`. Maybe in addition to consistency with naming we even need better naming itself. public int registerErrorCode(short errorCode) {

Reply:
changed to `erroGroupByCode`

Reply:
Unfortunately, I don't have any ideas how it can be improved right now. I will think about it in the background.